extensions/sql-database-projects/package.nls.json

```diff
@@ -19,10 +19,10 @@
 	"sqlDatabaseProjects.newItem": "Add Item...",
 	"sqlDatabaseProjects.newFolder": "Add Folder",
 
-	"sqlDatabaseProjects.importDatabase": "Import New Database Project",
 
 	"sqlDatabaseProjects.addDatabaseReference": "Add Database Reference",
 
 	"sqlDatabaseProjects.Settings": "Database Projects",
-	"sqlDatabaseProjects.netCoreInstallLocation": "Full Path to .Net Core SDK on the machine."
+	"sqlDatabaseProjects.netCoreInstallLocation": "Full Path to .Net Core SDK on the machine.",
+	"sqlDatabseProjects.welcome": "No open database projects currently open.\n[New Project](command:sqlDatabaseProjects.new)\n[Open Project](command:sqlDatabaseProjects.open)\n[Import Project from Database](command:sqlDatabaseProjects.importDatabase)"
```

Comment:
It sounds a little weird having open twice in the sentence.

Reply:
You right.  I'd wavered between "No open database projects" and "No database projects currently open".  Seems I picked both!